pkg/apis/policy/v1alpha1/types_certificaterequestpolicy.go

```diff
@@ -294,71 +301,67 @@ type CertificateRequestPolicyPluginData struct {
 // All selectors that have been configured must _all_ match a
 // CertificateRequest in order for the CertificateRequestPolicy to be chosen
 // for evaluation.
-// At least one of issuerRef or namespace must be defined.
+// At least one of IssuerRef or Namespace must be defined.
 type CertificateRequestPolicySelector struct {
-	// IssuerRef is used to match this CertificateRequestPolicy against processed
-	// CertificateRequests. This policy will only be evaluated against a
-	// CertificateRequest whose `spec.issuerRef` field matches
-	// `spec.selector.issuerRef`. CertificateRequests will not be processed on
-	// unmatched `issuerRef` if defined, regardless of whether the requestor is
-	// bound by RBAC.
-	// Accepts wildcards "*".
-	// Omitted values are equivalent to "*".
+	// IssuerRef is used to select on CertificateRequest issuer reference,
+	// meaning the CertificateRequestPolicy will only match on
+	// CertificateRequests that refer to matching issuers.
+	// CertificateRequests will not be processed on unmatched issuers,
+	// regardless of whether the requestor is bound by RBAC.
 	//
-	// The following value will match _all_ `issuerRefs`:
+	// The following value will match _all_ issuers:
 	// ```
 	// issuerRef: {}
 	// ```
 	// +optional
 	IssuerRef *CertificateRequestPolicySelectorIssuerRef `json:"issuerRef"`
 
-	// Namespace is used to select on Namespaces, meaning the
-	// CertificateRequestPolicy will only match on CertificateRequests that have
-	// been created in matching selected Namespaces.
-	// If this field is omitted, all Namespaces are selected.
+	// Namespace is used to select on namespace, meaning the
+	// CertificateRequestPolicy will only match on CertificateRequests that
+	// have been created in matching namespaces.
+	// If this field is omitted, all namespaces are selected.
 	// +optional
 	Namespace *CertificateRequestPolicySelectorNamespace `json:"namespace"`
 }
 
 // CertificateRequestPolicySelectorIssuerRef defines the selector for matching
-// on `issuerRef` of requests.
+// on issuer references of requests.
 type CertificateRequestPolicySelectorIssuerRef struct {
-	// Name is the wildcard selector to match the `spec.issuerRef.name` field on
-	// requests.
+	// Name is the wildcard selector to match the `spec.issuerRef.name` field
+	// on requests.
 	// Accepts wildcards "*".
```

Comment:
It feels like I need a thing that says `unlike the X which doesn't accept wildcards`.

Reply:
Is this correct?
```suggestion
	// Name is a wildcard enabled selector that matches the
	// `spec.issuerRef.name` field of requests.
	// Accepts wildcards "*".
```